sdk/keyvault/keyvault-common/README.md

```diff
@@ -0,0 +1,38 @@
+# Azure Key Vault Common client library for JavaScript
+
+An internal support library for the various Azure Key Vault client libraries.
+
+This package contains common code that needs to be shared among the other Azure Key Vault libraries. It is not published to NPM and is not meant for usage by any other consumers.
+
+Azure Key Vault libraries that use this package:
+
+- [@azure/keyvault-admin](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/keyvault/keyvault-admin/README.md)
+- [@azure/keyvault-certificates](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/keyvault/keyvault-certificates/README.md)
+- [@azure/keyvault-keys](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/keyvault/keyvault-keys/README.md)
+- [@azure/keyvault-secrets](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/keyvault/keyvault-secrets/README.md)
+
+## Getting started
+
+For information on getting started, please see our Key Vault client libraries.
+
+## Key concepts
+
+For information on key concepts, please see our Key Vault client libraries.
+
+## Examples
+
+For examples, please see our Key Vault client libraries.
+
+## Next steps
+
+For information on next steps, please see our Key Vault client libraries.
```

Comment:
Consider adding a link to the initial section where you have the list of the KV libraries with links?

Reply:
Thanks! added